front/components/assistant/AssistantDetails.tsx

```diff
@@ -78,7 +72,7 @@ export function AssistantDetails({
     includes: ["authors"],
   });
 
-  const effectiveAssistant = assistant ?? agentConfiguration;
```

Comment:
Let's remove this line?

Reply:
I kept effective assistant for now because the rebase with the AssistantDetails update PR would be painful => I intend to remove it in this PR directly if good for you